packages/monitor-v2/src/monitor-polymarket/README.md

```diff
@@ -29,3 +29,66 @@ All the configuration should be provided with following environment variables:
 - `RETRY_DELAY_MS` optionally specify a delay in milliseconds between retries, defaults to 0 delay.
 - `FILL_EVENTS_LOOKBACK_SECONDS` controls how many seconds of historic fills to consider for each proposal (defaults to `1800`).
 - `FILL_EVENTS_PROPOSAL_GAP_SECONDS` adds a positive buffer after the proposal block before fills are queried, ensuring we skip fills close to proposal timestamp (defaults to `300`).
+
+
+## Running the Polymarket Notifier Locally
+
+To run the Polymarket notifier on your local machine (outside serverless), follow these steps:
+
+### 1. Clone the required repositories
+
+If you haven’t already:
+
+- Bot configs: https://github.com/UMAprotocol/bot-configs
+- Protocol: https://github.com/UMAprotocol/protocol
+
+From any folder where you keep your projects, run:
+
+```bash
+    git clone https://github.com/UMAprotocol/bot-configs.git
+    git clone https://github.com/UMAprotocol/protocol.git
+```
+---
+
+### 2. Generate the `.env` file (inside the bot-configs repo)
+
+From the **root** of the `bot-configs` repository, run:
+
+```bash
+  node ./scripts/print-env-file.js ./serverless-bots/uma-config-5m.json polymarket-polygon-notifier \
+    | grep -Ev '^(SLACK_CONFIG|PAGER_DUTY_V2_CONFIG|DISCORD_CONFIG|DISCORD_TICKET_CONFIG|REDIS_URL|NODE_OPTIONS)=' \
+    > .env; printf 'LOCAL_NO_DATASTORE=true\nNODE_OPTIONS=--max-old-space-size=16000\n' >> .env
+```
+
+This command:
+
+- Prints the full environment configuration for the `polymarket-polygon-notifier`
+- Removes unnecessary configs (Slack, PagerDuty, Discord, Redis)
+- Appends:
+  - `LOCAL_NO_DATASTORE=true` → run without Google Cloud Datastore  
+  - `NODE_OPTIONS=--max-old-space-size=16000` → increase Node memory
+
+Now copy the absolute path of the generated `.env` file, e.g.:
+
+    $HOME/UMAprotocol/bot-configs/.env
+
+---
+
+### 3. Build the Protocol monitor package
+
+Navigate to the `monitor-v2` package in the **protocol** repository:
+
+    cd $HOME/UMAprotocol/protocol/packages/monitor-v2
+    yarn install
+    yarn build
+
+---
+
+### 4. Run the Polymarket notifier
+
+From the same `monitor-v2` directory, run the notifier pointing to the generated `.env` file:
+
+    DOTENV_CONFIG_PATH=$HOME/UMAprotocol/bot-configs/.env \
```

Comment:
People might have `.env` under `monitor-v2` from prior work, and it seems that is taking override over `DOTENV_CONFIG_PATH`